src/thunderbots/software/ai/passing/pass.h

```diff
@@ -0,0 +1,100 @@
+/**
+ * Declaration for the "Pass" Class
+ */
+
+#include <cstdlib>
+#include <array>
+#include <iostream>
+
+#include "geom/point.h"
+#include "util/timestamp.h"
+
+namespace AI::Passing {
+
+    // TODO: simple test for getters
+    /**
+     * This class represents a Pass, with a given start position, end position,
+     * speed, and start time
+     */
+    class Pass {
+    public:
+        /**
+         * Creates a pass with all 0 values
+         */
+        //Pass();
+
+        /**
+         * Create a pass with given parameters
+         *
+         * @param passer_point The point the pass should start at
+         * @param receiver_point The point the receiver should be at to receive the pass
+         * @param pass_speed_m_per_s The speed of the pass, in meters/second
+         * @param pass_start_time The time that the pass should start at
+         */
+        Pass(Point passer_point, Point receiver_point, double pass_speed_m_per_s, Timestamp pass_start_time);
+
+        /**
+         * Returns a reference to the receiver point
```

Comment:
The return types for the getters are no longer references. Comments should be updated (applies to the other functions in this file).

Reply:
donee